README.md

```diff
@@ -1015,7 +1015,56 @@ Width is not mandatory, but if it is not given then it uses the width of 12 auto
 
 **Note - newcol is deprecated**
 
-Older columns (`::newcol`) work, but they are deprecated. Column and row directives should be used instead.  
+Older columns (`::newcol`) work, but they are deprecated. Column and row directives should be used instead.
+
+## 18. Tabs
+
+Directive for adding tabbed content. It means that we can use tabs to separate content into different panels , allowing
```

Comment:
This intro could be rephrased, for example: 
 `aplus-tabs` is a directive for adding tabbed content. Tabs separate content into different panels so that one panel is displayed at a time.

Whitespace typo around the comma (but I think the whole comma should be removed): `panels , allowing`